resources/lib/services/library_updater.py

```diff
@@ -96,7 +96,7 @@ def update_kodi_library(self, data=None):
 
 
 def _compute_next_schedule():
-    update_frequency = g.ADDON.getSettingInt('auto_update')
+    update_frequency = g.ADDON.getSettingInt('auto_update') or None
```

Comment:
add a small note or the reason of "or None" reading the code in the future it is not clear why